packages/terra-framework-docs/src/terra-dev-site/test/collapsible-menu-view/PrePopulatedCollapsedMenuItems.test.jsx

```diff
@@ -1,26 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import IconSend from 'terra-icon/lib/icon/IconSend';
 import CollapsibleMenuView from 'terra-collapsible-menu-view';
 
-const CollapsibleMenuViewDemo = () => (
-  <CollapsibleMenuView alwaysCollapsedMenuItems={[
-    <CollapsibleMenuView.Item text="Collapsed Button 1" key="collapsedButton1" />,
-    <CollapsibleMenuView.Item text="Collapsed Button 2" key="collapsedButton2" />,
-    <CollapsibleMenuView.Item text="Collapsed Button 3" key="collapsedButton3" />]}
-  >
-    <CollapsibleMenuView.Item text="Button A" className="ButtonA" key="buttonA" />
-    <CollapsibleMenuView.Item text="Button B" className="ButtonB" key="buttonB" />
-    <CollapsibleMenuView.Divider key="Divider4" />
-    <CollapsibleMenuView.Item text="Button 1" className="Button1" key="button1" />
-    <CollapsibleMenuView.Item text="Button 2" className="Button2" key="button2" />
-    <CollapsibleMenuView.Item
-      icon={<IconSend />}
-      text="Button3"
-      className="IconOnly"
-      key="button3"
-      isIconOnly
-    />
-  </CollapsibleMenuView>
-);
+const CollapsibleMenuViewDemo = () => {
+  const [isSelectedButton, setIsSelectedButton] = useState(true);
```

Comment:
Can we set this to false initially so that we can test the selection functionality with a wdio test?

Reply:
Added https://github.com/cerner/terra-framework/pull/1909/commits/2b7641931bee0b9d69db41cbc10c15902e542a33